### PR TITLE
Adding a FAQ file and link on the README

### LINKED ISSUE #89: Add troubleshoot / configuration section
To have the best game experience it might be needed to do additional configuration. 

One example for that would be drawing over the notch area in android ([fix here](https://github.com/impulse/flutters/commit/25d4ce726cd18e426483e605fe3668ec68b3c12c )). 

It would be nice to have a collection of those configurations available in the documentation or in the readme.

### PR DESCRIPTION
**Pull request by luanpotter**
This fixes #89.

@erickzanardo I'm sure you have a lot of ideas of what to add to this file, so please let me know and I can improve it :)

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# FAQ
+
+This file contains Frequently Asked Questions and their answers. Please feel free to make PRs to amend this file with new questions.
+
+## Any audio/music/sounds related problems
+
+Flame only provides a thin wrapper over the [audioplayers](https://github.com/luanpotter/audioplayers) library. Please make extra sure the problem you are having is with Flame. If you have a low-level or hardware related audio problem, probably it's something related to audioplayers, so please feel free to head to that repository to search your problem or open your issue. If you indeed have a problem with the Flame wrap around audioplayers, please feel free to open an issue.
+
+## Drawing over the notch
```
Comment (erickzanardo) [inline FAQ.md]: Android only right? Maybe we should especify that

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+In order to draw over the notch, you must add the following line to your `styles.xml` file:
+
+```xml
+<item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
+```
+
+For more details, please check [this PR](https://github.com/impulse/flutters/commit/25d4ce726cd18e426483e605fe3668ec68b3c12c) from @impulse.
+
```
Comment (erickzanardo) [inline FAQ.md]: Add a section talking about out examples inside the docs folder

**Comment by erickzanardo** [comment]
Add a section on how to use the camera, you could just add a brief introduction and point to the there article I wrote on Fireslime's site

**Comment by erickzanardo** [comment]
Maybe a brief section to explain the difference between Game and BaseGame classes

**Comment by luanpotter** [comment]
It took way longer than I'd like to admit for me to figure out what you meant by camera hahaha

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+For more details, please check [this PR](https://github.com/impulse/flutters/commit/25d4ce726cd18e426483e605fe3668ec68b3c12c) from [impulse](https://github.com/impulse).
+
+## Examples
+
+We have a few examples! The simplest of all is in [the root example folder](/example/). This folder must be called `example` and contain a single project, so it's a very simple generic example game.
+
+However there is an [examples folder inside the docs](/doc/examples) that contains many more examples. Most newer features we are adding we are trying to add a specific example for that. Those are directed to specific features.
+
+If you want a more full-fledged game, please check [BGUG](https://github.com/fireslime/bgug), FireSlime's open source fast paced side-scrolling platformer. It is mostly up-to-date with Flame and uses a good chunck of the featues offered by the engine. Other examples game are [feroult](https://github.com/feroult)'s [Haunt](https://github.com/feroult/haunt), which is outdated but showcases Box2D, and also [renancaraujo](https://github.com/renancaraujo)'s port of the [trex](https://github.com/flame-engine/trex-flame) chrome game in Flame.
```
Comment (erickzanardo) [inline FAQ.md]: What about bob?

Comment (luanpotter) [reply FAQ.md]: sorry, I forgot

Comment (luanpotter) [reply FAQ.md]: had I had flame awesome this would not have happened hahaha

**Pull request merged**